src/title/Title.tsx

```diff
@@ -1,25 +1,32 @@
-import React, { Fragment, ReactNode } from 'react'
+import React, { ReactNode } from 'react'
 import cc from 'classcat'
 
+import { StyledTitle } from './Title.style'
+
 export interface TitleProps {
   readonly id?: string
   readonly className?: string
   readonly children: ReactNode
-  readonly headingLevel?: number | string
+  readonly headingLevel?: 1 | 2 | 3 | 4 | 5 | 6 | '1' | '2' | '3' | '4' | '5' | '6'
```

Comment:
nit: I would add a comment explaining that the string levels are there for retro-compatibility.